src/main/java/com/code_intelligence/jazzer/Jazzer.java

```diff
@@ -238,6 +287,26 @@ private static String appendWithPathListSeparator(String name, String... options
     return currentValue + File.pathSeparator + additionalOptions;
   }
 
+  private static Path findLibrary(List<String> candidateNames) {
+    if (isAndroid()) {
+      for (String candidateName : candidateNames) {
+        String candidateFullPath = "/apex/com.android.runtime/lib64/bionic/" + candidateName;
```

Comment:
Can we expect this path to be relatively stable?

Reply:
I only have access to Pixels, but across Pixels this path is consistent, and after talking with some of my teammates, everything under apex should be consistent across brands. 

Although, this is the hwasan fuzzing path, and this library will only be there on a hwasan Android builds, and as far as I know Google might be the only company currently providing hwasan builds.

https://developer.android.com/ndk/guides/hwasan
https://ci.android.com/builds/branches/aosp-master-with-phones-throttled/grid?
https://flash.android.com/

Reply:
However, even if you dont have a hwasan build, you can still fuzz Java

Reply:
Sounds good, thanks.